### DISCUSSION TITLE
What is the mpv equivalent of aplay -r 22050 -f S16_LE -t raw -?

### DISCUSSION THREAD

**Discussion by guest271314** [mpv-player/mpv#14666] [Q&A] [answered]
What is the `mpv` equivalent of `aplay -r 22050 -f S16_LE -t raw -`?

I tried 

```
mpv --no-video --audio-format=s16 --audio-channels=1 --audio-samplerate=22050 -
[file] Reading from stdin...
Failed to recognize file format.

Exiting... (Errors when loading file)
```

**Comment by yvvki** [comment, marked as answer]
Try this:
```
mpv --demuxer=rawaudio --demuxer-rawaudio-rate=22050 --demuxer-rawaudio-format=s16le --demuxer-rawaudio-channels=1 --no-terminal --keep-open=no -
```

**Reply by guest271314** [reply]
Occasionally clips at the end of playback on the version of `mpv` I tested mpv 0.34.1

FFmpeg library versions:
   libavutil       56.70.100
   libavcodec      58.134.100
   libavformat     58.76.100
   libswscale      5.9.100
   libavfilter     7.110.100
   libswresample   3.9.100
